### PR TITLE
feature: add port mapping for container

### PR DESCRIPTION
**Pull request by rudyfly**
### Ⅰ. Describe what this PR did
Add port mapping for container. You can use:
-p or --port to port binging into container
--expose to expose container's port to access.

### Ⅱ. Does this pull request fix one issue?

NONE

### Ⅳ. Describe how to verify it
```
# pouch run -d -p 8888:80 registry.hub.docker.com/library/httpd
```

### Ⅴ. Special notes for reviews

Signed-off-by: Rudy Zhang <rudyflyzhang@gmail.com>

### CODE REVIEW

File: test/cli_network_test.go [hunk 1]
```diff
@@ -283,3 +284,36 @@ func (suite *PouchNetworkSuite) TestNetworkCreateDup(c *check.C) {
...
+		"--name", funcname,
+		"-p", "9999:80",
+		image).Assert(c, icmd.Success)
+
+	time.Sleep(1 * time.Second)
+	err := icmd.RunCommand("curl", "localhost:9999").Compare(expct)
+	c.Assert(err, check.IsNil)
+
+	command.PouchRun("rm", "-f", funcname)
+}
```
Comment (Letty5411) [inline test/cli_network_test.go]: good test!

**Comment by Letty5411** [comment]
CI failed due to know issue #815

**Comment by Letty5411** [comment]
@rudyfly could you also add test to check the exposed port works?

File: cli/container.go [hunk 2]
```diff
@@ -131,18 +135,61 @@ func (c *container) config() (*types.ContainerCreateConfig, error) {
 		}
 	}
 
+	// parse port binding
```
Comment (allencloud) [inline cli/container.go]: In the future, we could encapsulate the following code block into a function like `parsePortMapping`

```
func parsePortMapping(ports []string)(ports [string]nats, portBindings xxxx)
```

**Comment by allencloud** [comment]
LGTM

**Pull request merged**